persistence.js

```diff
@@ -83,11 +89,7 @@ LevelPersistence.prototype.createRetainedStream = function (pattern) {
     }
   })
 
-  pump(this._db.createValueStream({
-    gt: 'retained:',
-    lt: 'retained\xff',
```

Comment:
I think this was actually a 'bug'. Technically the `lt` should've been `'retained:\xff'` (with a colon), compared to all other keys, e.g. `SUBSCRIPTIONS` etc.

Reply:
I think it is! :+1: